6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,21 @@
+
+const calculateAverage = (numbers) => {
+    const sum = numbers.reduce((acc, num) => acc + Number(num), 0);
```

Comment:
I don't think you need to convert `num` to a Number using the `Number()` function, since it is already a number
```suggestion
    const sum = numbers.reduce((acc, num) => acc + num, 0);
```